docs/installation-aws-ecs/docker-ecs-amundsen.yml

```diff
@@ -1,17 +1,16 @@
 version: '3'
 services:
   neo4j:
-      image: neo4j:3.3.0
+      image: neo4j:3.5.26
       container_name: neo4j_amundsen
       environment:
         - NEO4J_AUTH=neo4j/test
-        # These dont seem to be working though!
-        - NEO4J_dbms.active_database=amundsen.db
-        - NEO4J_dbms.directories.data=/neo4j/data
-        - NEO4J_dbms.directories.logs=/var/log/neo4j
-        - NEO4J_dbms.directories.import=/var/lib/neo4j/import
-        - NEO4J_dbms.security.auth_enabled=false
-        - NEO4J_dbms.connectors.default_listen_address=0.0.0.0
+        - NEO4J_dbms_active__database=amundsen.db
```

Comment:
I corrected the notation on these so they're read by Neo4j appropriately

Reply:
nice catch